app/environment.ts

```diff
@@ -1,4 +1,5 @@
-import { ChainID, StacksTestnet } from '@blockstack/stacks-transactions';
+import { ChainID } from '@stacks/transactions';
+import { StacksTestnet } from '@stacks/network';
```

Comment:
@yknl curious if these should be in the same package, eg network?

Reply:
I think it makes more sense to stay in the transaction package since that's the only place it's used internally.